examples/Resistor_Tutorial/plugs.py

```diff
@@ -0,0 +1,174 @@
+""" my_plugs.py """
+
+import time
+import random
+from typing import Tuple
+
+import pyvisa
+
+from openhtf.plugs import BasePlug
+
+
+### To use pyvisa ###
+# Use rm.list_resources() to find the devices plugged in
+# this will return a list of the connected devices such as:
+# ('USB0::0x1AB1::0x0E11::DP8C1234567890::INSTR',
+#   'ASRL1::INSTR', 'ASRL10::INSTR')
+#
+# Then run device = rm.open_resource(
+#     'USB0::0x1AB1::0x0E11::DP8C1234567890::INSTR'
+#     )
+# to connect to the device.
+# You can check the device ID with psu.query("*IDN?")
+
+class MultimeterPlug(BasePlug):
+    """Plug for control of a multimeter with pyvisa.
+
+    Args:
+        BasePlug (Plug): OpeHTF Plug object.
+    """
+    def __init__(self, simulate:bool = False)-> None:
+        self.simulate = simulate
+        self.logger.info("Logging mode is %s", self.simulate)
+        self.rm = None
+        self.multimeter = None
+
+    def connect(self)->None:
+        """Connect to the device using pysisa. 
+        Does nothing if using simulation mode.
+        """
+        if not self.simulate:
+            try:
+                self.rm = pyvisa.ResourceManager()
+                self.multimeter = self.rm.open_resource(
+                    "USB0::6833::8458::DM8A265201811::0::INSTR"
+                    )
+                self.logger.info("Connected to Multimeter")
+            except ConnectionRefusedError as error:
+                self.logger.warning(
+                    "Connection refused : %s, " \
```

Comment:
The trailing backslash `\` isn't need as Python will auto concat the two strings.